TeamCode/src/main/java/org/firstinspires/ftc/teamcode/systems/distance/DistanceSystem.java

```diff
@@ -25,15 +25,16 @@
     MecanumDriveSystem driveSystem;
     IMUSystem imu;
     ColorSystem colorSystem;
+    OpMode opMode;
 
     double initPitch;
     double initRoll;
 
-    public DistanceSystem(OpMode opMode, MecanumDriveSystem ds, IMUSystem iMu, ColorSystem cs) {
+    public DistanceSystem(OpMode opMode, MecanumDriveSystem ds, ColorSystem cs) {
```

Comment:
rename from ds to driveSystem and cs to ColorSystem. Also is DistanceSystem the best name for this class?

Reply:
nope.